### PR TITLE
Update `ES_HOST` to document TLS usage

### PR DESCRIPTION
**Pull request by ElanHasson**
Explain how to use ElasticSearch endpoints that use TLS.

### CODE REVIEW

**Comment by vmstan** [comment]
This should also probably be called out on https://docs.joinmastodon.org/admin/elasticsearch page as well.

**Comment by vmstan** [comment]
@ElanHasson can you please rebase?

**Comment by ElanHasson** [comment]
@vmstan should be all set. Feel free to change/add verbiage

File: content/en/admin/config.md [hunk 1]
```diff
@@ -387,7 +387,7 @@ See the [ElasticSearch setup page for details on each setting](../elasticsearch#
 
 #### `ES_HOST`
 
-Host of the Elasticsearch server. Defaults to `localhost`
+Host of the ElasticSearch server. Defaults to `localhost`. If using TLS, prepend the hostname with `https://`. For example: `https://myelastic.tld`.
```
Comment (vmstan) [inline content/en/admin/config.md:390]: Elasticsearch is proper usage. Also lets use example.com domain as proposed in #1227 for consistency.

```suggestion
Host of the Elasticsearch server. Defaults to `localhost`. If using TLS, prepend the hostname with `https://`. For example: `https://elastic.example.com`.
```

File: content/en/admin/elasticsearch.md [hunk 2]
```diff
@@ -71,6 +71,8 @@ ES_PRESET= # single_node_cluster, small_cluster or large_cluster
 # ES_PASS=
 ```
 
+_Note_: If using TLS, prepend the hostname with `https://`. For example: `https://myelastic.tld`.
```
Comment (vmstan) [inline content/en/admin/elasticsearch.md:74]: Same as above.

```suggestion
_Note_: If using TLS, prepend the hostname with `https://`. For example: `https://elastic.example.com`.
```

**Review by vmstan — CHANGES_REQUESTED** [review]

**Pull request merged**